### PR TITLE
silly changes

### PR DESCRIPTION
**Pull request by cbm755**

### CODE REVIEW

**Comment by lchar** [comment]
Boo!

File: notes-week-03/HOWTO-git.txt [hunk 1]
```diff
@@ -24,7 +24,6 @@ Get the "HTTPS" link from your webbrowser from the URL above.
 
 git clone <PASTE THE LINK HERE>
 
-for example:
```
Comment (cbm755) [inline notes-week-03/HOWTO-git.txt:27]: why did you remove this?

**Comment by patrickwalls** [comment]
Please resolve conflicts.

**Comment by cbm755** [comment]
nm
